schema.json

```diff
@@ -0,0 +1,579 @@
+      "email":"ishan.bhojwani@beta.gouv.fr",
+      "organisation":"transport.data.gouv.fr",
+      "role":"contributor"
+    },
+    {
+      "title":"Béatrice  Mercier",
+      "email":"beatrice.mercier@beta.gouv.fr",
+      "organisation":"transport.data.gouv.fr",
+      "role":"contributor"
+    },
+    {
+      "title":"Antoine Augusti",
+      "email":"antoine.augusti@data.gouv.fr",
+      "organisation":"Etalab",
+      "role":"contributor"
+    }
+  ],
+  "author":"transport.data.gouv.fr",
+  "contact":"contact@transport.beta.gouv.fr",
+  "fields":[
+    {
+      "name":"id_park",
+      "description":"L'identifiant unique du parking, délivré par le Point d’accès national. `INSEE-P-xxx` où `INSEE` est le code INSEE de la commune et `xxx` est le numéro d’ordre sur 3 chiffres.",
+      "example":"75114-P-001",
+      "type":"string",
+      "constraints":{
+        "required":true,
+        "pattern":"^([013-9]\\d|2[AB1-9])\\d{3}-P-\\d{3}$"
+      }
+    },
+    {
+      "name":"nom_park",
+      "description":"Nom du parking, ou nom donné dans son utilisation quotidienne en majuscules et sans accents. Recommandation : inutile de répéter le mot parking et ne pas dépasser les 64 caractères.",
+      "example":"REPUBLIQUE",
+      "type":"string",
+      "constraints":{
+        "required":true
+      }
+    },
+    {
+      "name":"insee",
+      "description":"Le code INSEE de la commune où le parking est localisé.",
+      "example":"75114",
+      "type":"string",
+      "constraints":{
+        "required":true,
+        "pattern":"^([013-9]\\d|2[AB1-9])\\d{3}$"
+      }
+    },
+    {
+      "name":"ad_park",
+      "description":"L'adresse de l’entrée principale du parking, suivi du code postal et du nom de la Commune (séparé par des virgules). Nomenclature pour les lieux proches des sorties d'autoroute ou de nationale : A11 Sortie 7 Le Mans Nord. Nomenclature pour les zones rurales sans adresse : \"Croisement de route_1 - route_2\" ou \"Le long de route_X après le passage à niveau\".",
+      "example":"3 rue de la Gare, 92300, Levallois-Peret",
+      "type":"string",
+      "constraints":{
+        "required":false
+      }
+    },
+    {
+      "name":"url",
+      "description":"Une adresse URL (Uniform Resource Locator) pointant vers une ressource disponible sur Internet où l'on peut obtenir d'autres informations pertinente relatives aux horaires d’ouverture et fermeture du parc, tarifs appliquées dans le parc, ressource disponible sur Internet où l'on peut réserver en ligne la place de parking.",
+      "example":"https://www.exemple.fr/stationnementrepublique/",
+      "type":"string",
+      "constraints":{
+        "required":false,
+        "format":"uri"
+      }
+    },
+    {
+      "name":"type_usag",
+      "description":"Type d'usagers autorisés à entrer dans le parc.",
+      "example":"tous_usagers",
+      "type":"string",
+      "constraints":{
+        "required":true,
+        "enum":[
+          "tous_usagers",
```

Comment:
hum, je suis pas un cador en orthographe mais ca ne devrait pas etre `tout_usagers` ?

Reply:
J'ai remplacé ceci par `tous` cc @be-mercier

Reply:
Corrigé dans https://github.com/etalab/schema-stationnement/pull/1/commits/a260ef6ad06feecea0227df783936c9a8bf77260